Source/Plugins/Core/com.equella.core/src/com/tle/core/settings/service/impl/ThemeSettingsServiceImpl.java

```diff
@@ -146,4 +157,63 @@ private String themeToJSONString(NewUITheme theme) throws JsonProcessingExceptio
     themeToString = objectMapper.writeValueAsString(theme);
     return themeToString;
   }
+
+  public String newUIToSass(NewUITheme theme) throws IOException {
```

Comment:
Should be private...

```suggestion
  private String newUIToSass(NewUITheme theme) throws IOException {
```

I also wonder of the name of this. Hmmm, needs more than just `newUI`... Maybe `newUIThemToSassVars`... Longish, but maybe warranted...

Actually, why not move this functionality to a method on the `NewUITheme` class. Maybe `toSassVars()`. Then done on line 201 you'd have:

```
fileSystemService.write(
        staging, SASS_VARS_FILENAME, new StringReader(getTheme().toSassVars()), false);
```

Reply:
Yeah, it makes more sense for it to be in the NewUITheme class, since it's more closely related to that